util/travis/clang-format-whitelist.txt

```diff
@@ -406,3 +406,5 @@ src/voxelalgorithms.h
 src/voxel.cpp
 src/voxel.h
 src/wieldmesh.cpp
+src/remoteplayer.h
+src/gui/guiConfirmRegistration.cpp
```

Comment:
this seems unrelated to this pr, why is this here?

Reply:
Because there was this line in Travis CI build:
"The file src/gui/guiConfirmRegistration.cpp is not compliant with the coding style:" - https://travis-ci.org/minetest/minetest/jobs/352884874